apps/server/src/modules/learnroom/controller/course.controller.ts

```diff
@@ -108,4 +108,17 @@ export class CourseController {
 	): Promise<void> {
 		await this.courseSyncUc.stopSynchronization(currentUser.userId, params.courseId);
 	}
+
+	@Get(':courseId/userPermissions')
```

Comment:
We should't use camelCase path fragments.

Reply:
I didn't know we have such policy. I changed it to `.../user-permissions`. Also the fronend got an update.